pennylane_orquestra/qiskit_device.py

```diff
@@ -46,22 +42,20 @@ class QeQiskitDevice(OrquestraDevice):
     qe_module_name = "qeqiskit.simulator"
     qe_function_name = "QiskitSimulator"
 
-    def __init__(self, wires, shots=10000, backend="qasm_simulator", **kwargs):
+    def __init__(self, wires, shots=1000, backend="qasm_simulator", **kwargs):
         if backend == "qasm_simulator":
-            if kwargs.get("analytic", None):
-                # Raise a warning if the analytic attribute was set to True
```

Comment:
Note: removed this and the following similar warning just for ease of maintainability